### PR TITLE
Showcase.ca

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Showcase.ca.xml [hunk 1]
```diff
@@ -1,65 +1,11 @@
...
-
-	¹ Secured by us
-
 -->
-<ruleset name="Showcase.ca (partial)">
-
+<ruleset name="Showcase.ca">
+	<target host="showcase.ca" />
+	<target host="www.showcase.ca" />
 	<target host="media.showcase.ca" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Showcase.ca.xml]: Also try:

```
assets.showcase.ca
uat.assets.showcase.ca
contests.showcase.ca
assets.contests.showcase.ca
uat.assets.contests.showcase.ca
l.showcase.ca
mail.showcase.ca
on.showcase.ca
preprod.showcase.ca
testsite.showcase.ca
tr.showcase.ca
```

Comment (ivysrono) [reply src/chrome/content/rules/Showcase.ca.xml]: I just try to clear and close #15916 ……

Comment (pipboy96) [reply src/chrome/content/rules/Showcase.ca.xml]: This should go into a follow-up PR then.

**Comment by pipboy96** [comment]
@ivysrono Make sure to not merge your own PRs, your push access may be revoked otherwise. I will request review from someone else in a moment.

**Comment by ivysrono** [comment]
@pipboy96 I know, PRs could not be merged by self.

**Comment by pipboy96** [comment]
@ivysrono I reminded just in case, don't worry.

**Comment by pipboy96** [comment]
@ivysrono Please try requesting review from @J0WI, for some reason I can't select them out of user list.

**Pull request merged**